ts/Core/Chart/StockChart.ts

```diff
@@ -1158,8 +1169,8 @@ function getDefaultAxisOptions(type: string, options: Highcharts.AxisOptions): H
  *
  * */
 
-(H as any).stockChart = stockChart;
-(H as any).StockChart = StockChart;
+H.stockChart = stockChart;
+H.StockChart = StockChart;
```

Comment:
This should be moved to the master file for stock (modules/stock). All files except masters have to work without Highcharts namespace and automatic modifications to become compatible with ES6 tree shaking (async). And it is okay to have any casts in the master file. No need to define the namespace there.